lib/Rex/Commands/File.pm

```diff
@@ -1112,14 +1112,17 @@ sub _append_or_update {
 
 =item extract($file [, %options])
 
-This function extracts a file. Supported formats are .box, .tar, .tar.gz, .tgz, .tar.Z, .tar.bz2, .tbz2, .zip, .gz, .bz2, .war, .jar.
+This function extracts a file. The target directory $to (if specified) will be created automatically
+if doesn't exist yet.
```

Comment:
I love the fact that you didn't forget about updating documentation! :+1: This line break seems a bit odd though.